WORKSPACE

```diff
@@ -57,21 +57,40 @@ load(
 
 container_repositories()
 
+container_pull(
```

Comment:
Move this to defs/container.bzl so that you only pull the base image when building another image

Reply:
The image is fetched lazily already, and since this is a repo rule, it needs to appear in the WORKSPACE file rather than in a BUILD file.